packages/dd-trace/src/ci-visibility/dynamic-instrumentation/worker/index.js

```diff
@@ -0,0 +1,88 @@
+const { parentPort } = require('worker_threads')
```

Comment:
No `'use strict'`?

Reply:
yeah I missed it, fixed in https://github.com/DataDog/dd-trace-js/pull/4826/commits/c92c40171e2f09cf928b2f7fd56457e0e2d9c18e